axon-framework/events/event-processors/streaming.md

```diff
@@ -587,6 +587,75 @@ public class AxonConfig {
 {% endtab %}
 {% endtabs %}
 
+#### Token Stealing
+
+As described at the [start](#tracking-tokens), streaming processor threads can "steal" tokens from one another.
+You may regard a token as stolen when a thread loses a [token claim](#token-claims).
```

Comment:
```suggestion
A token is "stolen" when a thread loses a [token claim](#token-claims).
```
... when a thread loses _its(?)_ [token claim](#token-claims).

Reply:
There's a chance to have multiple claims from the perspective of the PSEP, so I don't think _its_ works in that case, right?